dataminer/Functions/Dashboards_and_Low_Code_Apps/Low_Code_Apps/Style_guide.md

```diff
@@ -4,288 +4,210 @@ uid: LCA_Style_guide
+- Use the sidebar as the main navigation to the different pages defined in the app. When there are too many pages in your sidebar, or if you want to show the same data in another way, use the header bar as tabs to open hidden pages.
 
-  ![Form pop-up window](~/dataminer/images/LCA_Style_Guide_Form_Popup.gif)
+  > [!NOTE]
+  > When up to two components fill an entire page with no margins, you can place action buttons on the left side of the header bar. Just do not use them as tabs.
 
-- Every panel should have a close button in the top-right corner. This can be in the menu bar or it can be a separate button.
+- If you are using the header bar as tabs, make sure you **show a title on each page** or **use a circle icon that is filled** on the selected page.
 
-## Components
+- Place the "About" page **at the bottom** of the sidebar and **use the info circle icon**.
 
-### Titles
+- If your app has more than 10 pages (including sidebar and header tabs), **add an overview page**. It should highlight key data and provide quick links.
 
-- Titles are optional on pages, depending on whether users will need more context information about what they are looking at.
+## Panels
 
-- Avoid repeating the app name as a title on an app's pages.
+- Show **additional information in a side panel on the right** for each item on a page.
 
-- Do not use a background color for titles.
+- If your panel updates based on a selection, **add a title** with the selected item's name.
 
-- Do not make the titles too big: the focus should be on the content of the page.
+- Show **forms in a pop-up** panel.
 
-- We recommend adding a title to a page as a web component instead of just text. For example:
+- Always choose the **same width** in your actions if you are using the same panel more than once.
 
-  - HTML code:
+- Every panel should have a **close button in the top-right corner**. An icon is enough; the button does not need a label.
 
-    ```html
-    <div style="height:100%;display:flex;flex-direction:column;justify-content:center">
-      <H1 style="color: rgb(20,33,42); font-family:'Segoe UI Light','Segoe UI Web Light','Segoe UI Web Regular','Segoe UI',Arial,sans-serif;margin:0;font-weight:100;font-size:24px">
-        <b>People</b>
-      </H1>
-      <p style="color: rgb(20,33,42,0.7); font-family:'Segoe UI Light','Segoe UI Web Light','Segoe UI Web Regular','Segoe UI','Segoe UI Symbol',HelveticaNeue-Light,'Helvetica Neue',Arial,sans-serif;margin:0;font-weight:600;font-size:16px">
-      An overview of all managed people
-      </p>
-    </div>
-    ```
+- If you have too much information in one panel, you can split it into multiple panels and **use the header bar as tabs**.
 
-  - Result:
+- **Use the “As overlay” option** when opening a panel.
 
-    ![Good title](~/dataminer/images/LCA_Style_Guide_Title_Good.png)
+- Do not use panels as pages.
```

Comment:
Could you clarify this? How could panels be used as pages? Maybe we should rephrase this to a positive rule: what do you use panels for vs. what do you use pages for?

Reply:
So do you mean that a panel's width should never be set to 100%, because it should be clear that it is a panel instead of a page?